salesagility/SuiteCRM
# SuiteP Edit/Detail view issue with one field per row (task 229 issue #1931)

SuiteP Edit/Detail view issue with one field per row (task 229 issue #1931)

## Description
PR contains a fix for issue #1931

## Motivation and Context


## How To Test This
see instructions in related issue #1931 and change layouts of Edit/DetailView (in studio)

## Types of changes


### Final checklist
